www/js/components/BarChart.tsx

```diff
@@ -0,0 +1,201 @@
+
+import React, { useRef, useState } from 'react';
+import { array, string, bool } from 'prop-types';
+import { angularize } from '../angular-react-helper';
+import { View } from 'react-native';
+import { useTheme } from 'react-native-paper';
+import { Chart, CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend, TimeScale } from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+import Annotation, { AnnotationOptions } from 'chartjs-plugin-annotation';
+
+Chart.register(
+  CategoryScale,
+  LinearScale,
+  TimeScale,
+  BarElement,
+  Title,
+  Tooltip,
```

Comment:
They do! You can click the target and its tooltips show up, then click anywhere else to dismiss.

ChartJS considers the mobile interaction pretty well, which is another reason I chose instead of Plotly

Reply:
Cool! I was going to suggest that we might want to add the number at the end of the bar (like we do in the public dashboard) because sometimes you really want to get into the weeds, but this might be good enough.